src/common/components/drafts/index.tsx

```diff
@@ -249,13 +264,30 @@ export class Drafts extends BaseComponent<Props, State> {
     onHide();
   };
 
+  clone = (item: Draft) => {
+    this.setState({ isClone: true });
+    const { activeUser } = this.props;
+    const { title, body, tags, meta } = item;
+    const cloneTitle = _t("g.copy") + " " + title;
+    const draftMeta: DraftMetadata = meta!;
+    addDraft(activeUser?.username!, cloneTitle, body, tags, draftMeta).then((resp) => {
+      this.stateSet({ list: this.sort(resp.drafts), isClone: false }, this.handleScroll);
+      success(_t("g.clone-success"));
+    });
+  };
```

Comment:
1. Better to use it as async function instead of `then` construction
2. Should you set isClone to false in finally of promise?